### PR TITLE
fix conversion from decimal string into oct

### PR DESCRIPTION
**Pull request by 6293**

### CODE REVIEW

File: apfloat/src/ieee.rs [hunk 1]
```diff
@@ -1971,14 +1971,16 @@ impl<S: Semantics> IeeeFloat<S> {
         //       42039/12655 < L < 28738/8651  [ numerator <= 65536 ]
 
         // Check for MAX_EXP.
-        if normalized_exp.saturating_sub(1).saturating_mul(42039) >= 12655 * S::MAX_EXP as i32 {
+        if normalized_exp.saturating_sub(1).saturating_mul(42039) as i64
+            >= 12655 * S::MAX_EXP as i64
```
Comment (6293) [inline apfloat/src/ieee.rs:1975]: I forgot the fact that MAX_EXP in octal is 262143 so this multiplication was unmanageable for i32.

Comment (dr-orlovsky) [reply apfloat/src/ieee.rs:1975]: Thanks for spotting it!

**Review by dr-orlovsky — APPROVED** [review]
ACK d99d7bd6275db65099a8a988457e7176cc243174

I assume I have to release a new version?

**Comment by dr-orlovsky** [comment]
New version of `amplify_apfloat` is released

**Pull request merged**